Tasks/AzureResourceGroupDeployment/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 2,
     "Minor": 2,
-    "Patch": 4
+    "Patch": 6
```

Comment:
122 is at version 5. Ayantika will change 122 also to 6. I am changing this to 5.